app/helpers/ems_infra_helper/textual_summary.rb

```diff
@@ -127,6 +127,12 @@ def textual_used_availability_zones
                  :link => ems_infra_path(@record.id, :display => 'availability_zones'))
   end
 
+  def textual_ems_cloud
+    return nil if !@record.provider.respond_to?(:cloud_ems)
+
+    textual_link(@record.try(:provider).try(:cloud_ems).first)
```

Comment:
@aufi Thanks for the comment!  I think you only get 'nil' if the provider doesn't respond to :cloud_ems, which I check for... ?